sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -201,7 +209,8 @@ def check_classification_synthetic(presort, loss):
 
 
 def test_classification_synthetic():
-    for presort, loss in product(('auto', True, False), ('deviance', 'exponential')):
+    for presort, loss in product(('auto', True, False),
```

Comment:
Please refrain from making changes that are not related to the main topic of your PR. This adds noise and makes the PR review less pleasant for everyone involved.